test/check-machines-disks

```diff
@@ -82,7 +82,11 @@ class TestMachinesDisks(machineslib.VirtualMachinesCase):
             else:
                 b.wait_not_present(f"#vm-{name}-disks-{target}-edit-dialog")
 
-        self.createVm("subVmTest1")
+        if "rhel-8" in m.image:
+            # q35 in rhel-8 can't hotplug
+            self.createVm("subVmTest1", os="linux2016")
```

Comment:
This is done several times -- perhaps factor this out into setUp(), create some `create_options` dict (empty by default, that option on rhel8), and call `self.createVm("subVmTest1", **create_options)`?

That should also apply to centos-8-stream, no?

Reply:
Yep. I might try sticking with linux2016 on rhel-8 in general, and then bumping it up to linux2022 in specific tests explicitly.

Reply:
Oh, this just works.

Reply:
> That should also apply to centos-8-stream, no?

We don't test on centos-8-stream here, do we?

Reply:
Right, that overlapped with yesterday's c8s cleanup.